full/src/main/java/apoc/date/DateExpiry.java

```diff
@@ -17,15 +17,15 @@
  */
 @Extended
 public class DateExpiry {
-	@Procedure(mode = Mode.WRITE, deprecatedBy = "apoc.ttl.expireAtInstant")
+	@Procedure(mode = Mode.WRITE, deprecatedBy = "apoc.ttl.expire")
```

Comment:
non possiamo cambiare il nome della procedura devi cambiare quello in description